src/test/bin/winkernel/control.cpp

```diff
@@ -884,6 +884,10 @@ QuicTestCtlEvtIoDeviceControl(
                 Params->Family));
         break;
 
+    case IOCTL_QUIC_RUN_VALIDATE_SET_PARAM:
+        QuicTestCtlRun(QuicTestSetParam());
+        break;
+
```

Comment:
Move this to the end too since it's value is updating to the end (if you follow my other comment).